src/app/components/cards/Comment.scss

```diff
@@ -56,6 +59,7 @@
     .Voting {
       margin-left: 1rem;
       border-right: none;
+      display: none;
```

Comment:
In long nested comments, the width is too small and the vote/downvote buttons and the count break the layout and make it two lines when collapsed. Does it make sense to vote/downvote a comment without reading it anyway?

Reply:
Let's leave it for now and check how to best address it later